.gitignore

```diff
@@ -80,7 +80,9 @@ target/
 # IDEs
 .idea
 tags
-.vscode
+.vscode/*
+!.vscode/extensions.json
```

Comment:
Add .vscode/extensions.json to prompt developers to install recommended VSCode extenions